php-fpm/Dockerfile

```diff
@@ -278,9 +278,10 @@ RUN if [ ${INSTALL_XHPROF} = true ]; then \
     ) \
     && rm -r xhprof \
     && rm /tmp/xhprof.tar.gz \
+    && COPY ./xhprof.ini /usr/local/etc/php/conf.d \
```

Comment:
I believe @hackel  is correct and the correct solution would be to leave the `COPY` statement outside the `if` and change this line to the following:
```
&& echo extension=tideways.so > /usr/local/etc/php/conf.d/php-ext-tideways.ini \
```

You will also want to comment out (or remove) `extension=tideways.so` from `php-fpm/xhprof.ini`.